it/src/test/java/com/linecorp/centraldogma/it/WatchTest.java

```diff
@@ -79,14 +78,19 @@ public void testWatchRepository() throws Exception {
         final Revision rev2 = commit.revision();
 
         assertThat(rev2).isEqualTo(rev1.forward(1));
-        assertThat(future.get(3, TimeUnit.SECONDS)).isEqualTo(rev2);
+        ExpectedExceptionAppender.withExpectedException(
+                "AnnotatedConnectException", "Connection refused: /127.0.0.1:", () ->
```

Comment:
'Connection refused' and IP address are platform-dependent strings. Could we only check the exception type?